staging/src/k8s.io/client-go/discovery/cached/memcache.go

```diff
@@ -131,7 +132,7 @@ func (d *memCacheClient) Invalidate() {
 		for _, v := range g.Versions {
 			r, err := d.delegate.ServerResourcesForGroupVersion(v.GroupVersion)
 			if err != nil || len(r.APIResources) == 0 {
-				utilruntime.HandleError(fmt.Errorf("couldn't get resource list for %v: %v", v.GroupVersion, err))
+				glog.Errorf("couldn't get resource list for %v: %v", v.GroupVersion, err)
```

Comment:
one can override `runtime.ErrorHandlers`. With this change one is forced into glog.

Reply:
agree, don't import glog.

This error is only important if it happens every time. You might consider e.g. counting occurrances per groupversion and only log an error after three and then at 10, 50, 100, 1000... occurrences. And clear the count after every success.

Alternatively, make a version of HandleError that makes it clear a call stack won't make it easier to diagnose.